projects/sw-offline-google-analytics/test/unit/replay-queued-requests.js

```diff
@@ -38,17 +39,24 @@ const testLogic = (initialUrls, expectedUrls, time, additionalParameters) => {
 
 describe('replay-queued-requests', () => {
   const urlPrefix = 'https://replay-queued-requests.com/';
+  const initialTimestamp = 1470405670000;
```

Comment:
It's an arbitrary 📅  from earlier this morning, but the exact value doesn't matter in the context of the test. I'll put in a comment to that effect.